build.py

```diff
@@ -96,10 +96,14 @@ def __init__(self, name, env):
     def add_run_test_builds(self):
         # For everything that gets installed to build/unittests, add a rule for +basename
         # that runs the test from its original location.
+        unittest_dir = 'build/unittests/'
+        if self.globalEnv.TargetOSIs('windows'):
```

Comment:
I've been trying to avoid branches like this for path handling. How about `os.path.join('build', 'unittests', '')`?